examples/dataservice-read/example.cpp

```diff
@@ -129,8 +129,15 @@ int RunExample() {
   std::shared_ptr<olp::http::Network> http_client = olp::client::
       OlpClientSettingsFactory::CreateDefaultNetworkRequestHandler();
 
-  // Initialize authentication settings
-  olp::authentication::Settings settings({kKeyId, kKeySecret});
+  // Get your access credentials from the **credentials.properties** file that
+  // you can download from the Open Location Platform (OLP)
+  // [website](https://developer.here.com/olp/documentation/access-control/user-guide/topics/get-credentials.html).
```

Comment:
As this is not a Doxygen comment it doesn't make much sense to use Doxygen formatting.
Just a hint for the future.